templates/webapi/main/parent_group_overview.html.ep

```diff
@@ -15,27 +15,39 @@
         </button>
     </form>
 % }
-<h2>
+<h2 class="mb-4">
     Last Builds for <%= $group->{name} %>
 </h2>
+
+
+%= include 'main/pinned_comments'
+
+<ul class="nav nav-pills">
+  <li class="nav-item">
+    <a class="nav-link active" id="pills-home-tab" data-toggle="pill" href="#pills-home" role="tab" aria-controls="pills-home" aria-selected="true">Grouped by build</a>
+  </li>
+  <li class="nav-item">
+    <a class="nav-link" id="pills-profile-tab" data-toggle="pill" href="#pills-profile" role="tab" aria-controls="pills-profile" aria-selected="false">Grouped by group</a>
+  </li>
+</ul>
+
+
 <div class="well well-lg" id="group_description">
     % if (my $description = $group->{rendered_description}) {
         %= $description
     % }
-    <div id="child_groups">
-        <h3>Accumulated results for</h3>
-        <ul>
-        % for my $child_group (@$child_groups) {
-            <li>
-                %= link_to $child_group->name => url_for('group_overview', groupid => $child_group->id)
-            </l>
-        % }
-        </ul>
-    </div>
 </div>
-%= include 'main/pinned_comments'
 
 <div id="build-results"></div>
-%= include 'main/group_builds', build_results => $build_results, group => $group, children => $children, default_expanded => 1
+
+<div class="tab-content" id="pills-tabContent">
+  <div class="tab-pane fade show active" id="pills-home" role="tabpanel" aria-labelledby="pills-home-tab">
+    %= include 'main/group_builds', build_results => $build_results, group => $group, children => $children, default_expanded => 1
+  </div>
+  <div class="tab-pane fade" id="pills-profile" role="tabpanel" aria-labelledby="pills-profile-tab">
+    %= include 'main/group_builds2', build_results => $build_results, group => $group, children => $children, default_expanded => 1
```

Comment:
I know finding good variable and file names is hard but I consider it very important as it already shows if we have a good design idea. "group_builds2" is certainly not the best pick :D I assume this is the alternative to "group_builds" but you are for now including both here. I don't understand the intention.

Reply:
@okurz Thanks for the tips.
The reason I maintained both is to allow user to select what format to see using a selector. I think could be interesting to have more options to see the same data. Some people used to use the original version and maybe is not interested in the new one. Additionally painting both views is not overloading the server processing because the data is the same in both cases. 

Of course this is an open for discussion, and I would like to have several opinions about that

For now this is a draft proposal, still working on that.

Reply:
Of course. This is also what we discussed yesterday in the meeting: Have the new view as an option but not as new default. We mentioned though that the selection for the new view option should be done in the job group settings where we also allow to select the sort order as a third option

Reply:
It still is not in this commit. Taking care of it

Reply:
@okurz There is a problem setting up a third option. The variable "job_group_parents" where is stored, is a boolean. I am working in an other settings entry where you could chose what is the default view you want to see.
A second option, could be to change the type of this field from boolean to integer. But It could generate side effects, and I think the better is the first idea. 
What do you think?

Reply:
This functionality is ready for review